Composer/SearchScriptHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Composer;
+
+use Composer\Script\CommandEvent;
+use Sensio\Bundle\DistributionBundle\Composer\ScriptHandler;
+
+class SearchScriptHandler extends ScriptHandler
+{
+    /**
+     * @param $event CommandEvent A instance
+     */
+    public static function initBundle(CommandEvent $event)
+    {
+        $options = parent::getOptions($event);
+        $consoleDir = isset($options['symfony-bin-dir']) ? $options['symfony-bin-dir'] : $options['symfony-app-dir'];
```

Comment:
The problem is that this would only work with a symfony-standard based project. 

We also have an implicit dependency on the `DistributionBundle` here which is bad.